transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -1299,9 +1299,10 @@ private static void assertGroupAndInvoker(EventExecutorGroup group, ChannelHandl
     static final class TailContext extends AbstractChannelHandlerContext implements ChannelInboundHandler {
 
         private static final String TAIL_NAME = generateName0(TailContext.class);
-
+        private final AbstractChannel.AbstractUnsafe unsafe;
```

Comment:
I'm not sure if we need a field for this. Can we just add a private function that returns an AbstractUnsafe?

```java
private AbstractUnsafe unsafe() {
    return (AbstractUnsafe) pipeline().channel().unsafe();
}